src/tag-input/src/TagInput.js

```diff
@@ -166,30 +176,118 @@ const TagInput = memo(
+            items={hasAutocomplete ? autocompleteItems : []}
+            id={inputId}
+            selectedItem=""
+            inputValue={inputValue}
+          >
+            {autocompleteProps => {
+              const {
+                closeMenu,
+                getInputProps,
+                getRef: autocompleteGetRef,
+                getToggleButtonProps,
+                highlightedIndex
+              } = autocompleteProps
+
+              const {
+                onBlur: autocompleteOnBlur,
+                onChange: autocompleteOnChange,
+                onKeyDown: autocompleteKeyDown,
+                ...autocompleteRestProps
+              } = getInputProps()
+
+              return (
+                <>
+                  <TextInput
+                    appearance="none"
+                    disabled={disabled}
+                    height={height - 4}
+                    width="auto"
+                    type="text"
+                    {...inputProps}
+                    {...autocompleteRestProps}
+                    value={inputValue}
+                    id={inputId}
+                    ref={textInputRef => {
+                      autocompleteGetRef(textInputRef)
+                      if (inputRef instanceof Function) {
+                        inputRef(textInputRef)
+                      } else if (inputRef) {
+                        inputRef.current = textInputRef
+                      }
+                    }}
+                    onBlur={e => {
+                      autocompleteOnBlur(e)
+                      if (inputProps.onBlur) {
+                        safeInvoke(inputProps.onBlur, e)
+                      }
+                    }}
+                    onFocus={e => {
+                      handleInputFocus(e)
+                      if (inputProps.onFocus) {
+                        safeInvoke(inputProps.onFocus, e)
+                      }
+                    }}
+                    onChange={e => {
+                      handleInputChange(e)
+                      autocompleteOnChange(e)
+                    }}
+                    onKeyDown={e => {
+                      autocompleteKeyDown(e)
+                      if (e.key === 'Backspace' || !(highlightedIndex > -1)) {
+                        handleKeyDown(e)
+                        if (e.key === GET_KEY_FOR_TAG_DELIMITER[tagSubmitKey]) {
+                          closeMenu()
+                          setInputValue('')
+                        }
+                      }
+                      if (e.key === 'Backspace' && e.target.selectionEnd === 0) {
+                        closeMenu()
+                      }
+                    }}
+                  />
+                  {hasAutocomplete && (
+                    <Pane
+                      as="button"
+                      background="gray100"
+                      position="absolute"
+                      top={6}
+                      right={4}
+                      height={20}
```

Comment:
How do you feel about using `majorScale`/`minorScale` here for these?

Reply:
Sure, I can do that